app/routes/index/chapters/chapter/questions/question.js

```diff
@@ -175,14 +175,7 @@ export default Ember.Route.extend({
     saveTags(member) {
```

Comment:
Remove `member` from args. Throws error "'member' is defined but never used." As mentioned in the TODO comment, this whole route action may no longer be needed, but remove the member form the args to remove the error.